sdp/processors/datasets/coraa/create_initial_manifest.py

```diff
@@ -0,0 +1,117 @@
+import glob
+import os
+from pathlib import Path
+from typing import List
+from huggingface_hub import snapshot_download
+import pandas as pd
+
+import rarfile  #Needs to be installed
+import sox
+from sox import Transformer
+
+from sdp.processors.base_processor import BaseParallelProcessor, DataEntry
+from sdp.utils.common import extract_archive
+
+class CreateInitialManifestCORAA(BaseParallelProcessor):
```

Comment:
Let's also add this processor to the list of "Dataset-specific processors" in the docs, since we say  "Here is the full list of all available processors and their supported arguments".

Reply:
Done